test/integration/preview_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class PreviewTest < Capybara::Rails::TestCase
+
+	setup do
+		signs(:default).update(name: 'Ross Sign')
+    	signs(:default).slides.clear
+		users(:ross).signs << signs(:default)
+	end
+
+	scenario "notifications have slide preview for sign owners" do 
+		sign_in users(:james)
```

Comment:
Does this work? I thought devise sign-in didn't work for integration tests. Am I think of a different app?

You should be able to use a controller test if your arrange everything correctly.

If this does work, can you add an assert that confirms the user is logged in?